pages/docs/tutorials/native/mpp-ios-android.md

```diff
@@ -411,29 +391,96 @@ SharedCode/build/bin/iOSx64/main/release/framework/SharedCode.framework
 SharedCode/build/bin/iOS/main/release/framework/SharedCode.framework
 ```
 
-It is only possible to use one of those binaries in the project to target either
-iOS emulator (x86_64) or the device (arm64), debug or release. You may to create
-[an universal framework](https://medium.com/swiftindia/build-a-custom-universal-framework-on-ios-swift-549c084de7c8)
-from those two frameworks to simplify local development.
-Let's focus on the emulator for now.
+### Tuning the Gradle Build Script
+We need to supply the right Framework depending on the selected target in the Xcode
+project. It also depends from the `Release` or `Debug` configuration. Lastly,
+we'd like to make Xcode to compile the Framework for us before the build.
+We need to include the additional task to the end of the `SharedCode/build.gradle` file:
+
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-highlight-only="1" auto-indent="false">
+
+```groovy
+task packForXCode(type: Sync) {
+    final File frameworkDir = new File(buildDir, "xcode-frameworks")
+    final String mode = System.getenv('CONFIGURATION')?.toUpperCase() ?: 'DEBUG'
+    final String target = System.getenv('SDK_NAME')?.startsWith("iphoneos") ? 'iOS' : 'iOSx64'
+
+    inputs.property "target", target
+    inputs.property "mode", mode
+    outputs.dir frameworkDir
```

Comment:
Is `outputs.dir` required to be set explicitly for `Sync` task?

Reply:
My bad. Removed